sbol_utilities/package.py

```diff
@@ -64,107 +65,186 @@ def regularize_package_directory(dir: str):
         raise ValueError(f'Package {dir}: {PACKAGE_DIRECTORY} subdirectory should not have any subdirectories of its '
                          f'own, but found {package_sub_dirs[0]}')
 
-def define_package(doc: sbol3.Document):
-    """Function to take an sbol document, check if it is a package, and create
-    a new sbol document with the package definition included
+def aggregate_subpackages(root_package_file: sbol3.Document,
+                         *sub_package_files: sbol3.Document):
+    """Function to take one or more sbol documents, check if they are a package,
+     and create a new sbol document with the package definition included
 
     Args:
-        doc (sbol3.Document): The document to be checked for a package
+        root_package_file (sbol3.Document): First document, for the package
+        *subpackage_files (sbol3.Document): The document(s) for subpackages
 
     Return
-        doc (sbol3.Document): The document with the added package info
+        package: The package definition
     """
-    # Call check_namespace
-    logging.info('Checking namespaces')
-    is_package, package_namespace = check_namespaces(doc)
-    logging.info(f'SBOL Document is a package: {is_package}')
+    # Put all the input files in one list
+    # Means I am treating the root and subpackage files exactly the same
+    # CHECK: How should I be treating the root and the sub packages differently?
```

Comment:
Hmm... that probably does need clarification. It's specifically talking about the case when the package is a directory. In that case, it wouldn't have members because there's not file, just a directory containing sub-package files. Do you want to suggest a change and have me review, or vice versa?

Reply:
I think it would be best if you made the change- I feel like still do not have a firm grasp on working with a directory rather than with files.

Reply:
Made the change: https://github.com/SynBioDex/SEPs/commit/5faba2733ae87c83ebe09fb70c3248da2f7583cf
Please let me know if that makes more sense to you.